src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -286,5 +286,19 @@ public void Embed()
             csc.EmbeddedFiles = MSBuildUtil.CreateTaskItems();
             Assert.Equal(@"/debug:portable /out:test.exe test.cs", csc.GenerateResponseFileContents());
         }
+
+        [Fact]
+        public void EditorConfig()
```

Comment:
:bulb: A new test for a subdirectory containing a space would be good.

:bulb: A new test covering a case where the path contains `..\` would be good.

:bulb: Tests covering multiple **.editorconfig** files would be good.